### PR TITLE
influx CLI added

### LINKED ISSUE #382: [CLI] influxdb v1
Add the influxdb CLI to `arkade get` 

Add the V1 cli https://github.com/influxdata/influxdb/releases from here
## Context

We are adding the influx helm chart, so would make sense to add the cli.

### PR DESCRIPTION
**Pull request by developer-guy**
Signed-off-by: Batuhan Apaydın <batuhan.apaydin@trendyol.com>

## Description
influx CLI added

## Motivation and Context

Fixes #382 
- [x] I have raised an issue to propose this change ([required](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md))

## How Has This Been Tested?

```bash
$ docker container run --rm -v $(pwd):/workspace -w /workspace -ti golang:1.16.3-alpine sh
# go build
# ./arkade get influx
```
Screen Shot 2021-07-06 at 14 29 01

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

- [ ] I have tested this on arm, or have added code to prevent deployment

### CODE REVIEW

**Comment by developer-guy** [comment]
> Weren't we going to add v1? There's a binary, you just have to scroll back

Couldn't find the client version v1 😔

**Comment by developer-guy** [comment]
Folks @Waterdrips @alexellis do you have any suggestions, what should I do ?

**Comment by developer-guy** [comment]
> Folks @Waterdrips @alexellis do you have any suggestions, what should I do ?

Any update folks, @alexellis @Waterdrips

**Comment by alexellis** [comment]
This needs a rebase please

**Comment by developer-guy** [comment]
> This needs a rebase please

done.

**Review by Waterdrips — APPROVED** [review]

**Comment by alexellis** [comment]
@developer-guy you haven't answered my question about the v1 binary. The release is present, you just need to navigate back several pages to find it.

In this case, if you can rebase I will merge and see if there is demand for v1.

Alex

**Comment by Waterdrips** [comment]
https://github.com/influxdata/influxdb/releases/tag/v1.8.6
looks like the cli is in the relevant `OSS BINARY FILES` entry

**Comment by developer-guy** [comment]
> https://github.com/influxdata/influxdb/releases/tag/v1.8.6
> looks like the cli is in the relevant `OSS BINARY FILES` entry

@Waterdrips thank you so much for this reply, I updated the version as 1.8.6, would you like you review it again this PR, thank you 🙏

**Comment by developer-guy** [comment]
@alexellis @Waterdrips IMHO, we can merge and close this one.

File: pkg/get/tools.go [hunk 1]
```diff
@@ -1168,6 +1168,24 @@ https://github.com/{{.Owner}}/{{.Repo}}/releases/download/{{.Version}}/{{.Name}}
...
+		{{ if eq .Arch "x86_64" -}}
+		{{$arch = "amd64"}}
+		{{- end -}}
+
+		{{$ext := ".tar.gz"}}
+		{{ if HasPrefix .OS "ming" -}}
+		{{$ext = ".zip"}}
+		{{- end -}}
+
+				https://dl.{{.Owner}}.com/{{.Repo}}/releases/{{.Repo}}_client_{{.Version}}_{{.OS}}_{{$arch}}{{$ext}}`,
```
Comment (alexellis) [inline pkg/get/tools.go:1187]: I was sure that I downloaded this from GitHub releases.

**Comment by alexellis** [comment]
Let's revert back to the original 2.0 version that you proposed. It should also be download-able from GitHub correct?

**Comment by developer-guy** [comment]
> Let's revert back to the original 2.0 version that you proposed. It should also be download-able from GitHub correct?
I updated the code according to your reviews @alexellis, I couldn't find the GitHub releases download link btw.

**Comment by alexellis** [comment]
They seem to have dropped ARM support in 2.0 :thinking:

**Comment by alexellis** [comment]
It seems like CI is failing, can you look into it?

**Comment by alexellis** [comment]
Thank you for the patience on this PR :)

**Comment by developer-guy** [comment]
> Thank you for the patience on this PR :)

Thank you so much for merging 🙏

**Pull request merged**